src/openfermion/hamiltonians/_jellium_test.py

```diff
@@ -22,6 +22,51 @@
 from openfermion.utils import count_qubits, eigenspectrum, Grid, is_hermitian
 
 
+class WignerSeitzRadiusTest(unittest.TestCase):
```

Comment:
You took these from `_plane_wave_hamiltonian_test.py`, right? Delete them there.